content/softwares/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Softwares"  # Add a page title.
```

Comment:
Maybe we should use **software** instead of **softwares**. See https://forum.wordreference.com/threads/does-software-have-a-plural-form.231221/